Adds various new configs and keybind options to stonesense (#134)
DFHack/stonesense: docs/changelog.txt

# Future

## New Features
- `stonesense`: added config for closeOnEsc to disable closing stonesense with the escape key

> - `stonesense`: added configs to show creature moods and jobs by default

Reviewer: ```suggestion
- `stonesense`: added configs for whether we show creature moods and jobs at startup
```

> - `stonesense`: added configs to show creature moods and jobs by default

Reviewer: ```suggestion
- `stonesense`: added configs to show creature moods and jobs by default
```
```suggestion
- `stonesense`: added configs for whether creature moods and jobs are displayed
```
